tests/sles4sap/hana_install.pm

```diff
@@ -190,6 +190,7 @@ sub run {
     my $cmd = join(' ', $hdblcm, @hdblcm_args);
     record_info 'hdblcm command', $cmd;
     assert_script_run $cmd, $tout;
+    type_string "cd\n";    # Back to root directory
```

Comment:
I think it would be better to move this to `lib/sles4sap::copy_media()` as it's there where the current directory is being changed. I don't believe `hdblcm` uses anything from a relative path.

Reply:
Could be yes, I will check and test.

Reply:
It seems that NW installer doesn't like to be executed if we run it from another directory: http://1b210.qa.suse.de/tests/8868#step/netweaver_install/45

Reply:
Done!

Reply:
The latest change I made needs some modification in the jobs group YAML file, because I included the ARCH in the mount command.

I will make a MR for the BV and the maintenance tests before merging this PR.